examples/platforms/virtual/platform.c

```diff
@@ -80,13 +81,21 @@ void PlatformProcessDrivers(otInstance *aInstance)
     struct timeval timeout;
     int rval;
 
+    if (!UartInitialized)
+    {
+        UartInitialized = true;
+        otPlatUartEnable();
+    }
+
     FD_ZERO(&read_fds);
     FD_ZERO(&write_fds);
     FD_ZERO(&error_fds);
 
-    posixUartUpdateFdSet(&read_fds, &write_fds, &error_fds, &max_fd);
-    posixRadioUpdateFdSet(&read_fds, &write_fds, &max_fd);
-    posixAlarmUpdateTimeout(&timeout);
+#ifndef _WIN32
```

Comment:
How about something like this instead: `#ifndef HAVE_SELECT` instead? Would need to add the following line to `configure.ac`:

```
AC_CHECK_FUNCS([select])
```